biome.jsonc

```diff
@@ -1,20 +1,22 @@
 {
-	"$schema": "https://biomejs.dev/schemas/1.6.4/schema.json",
-	"organizeImports": {
-		"enabled": true
-	},
-	"linter": {
-		"enabled": true,
-		"rules": {
-			"recommended": true
-		}
-	},
-	"formatter": {
-		"enabled": true
-	},
-	"json": {
-		"formatter": {
-			"enabled": true
-		}
-	}
+  "$schema": "https://biomejs.dev/schemas/1.6.4/schema.json",
+  "organizeImports": {
+    "enabled": true
+  },
+  "linter": {
+    "enabled": true,
+    "rules": {
+      "recommended": true
+    }
+  },
+  "formatter": {
+    "enabled": true,
+    "indentWidth": 2,
+    "indentStyle": "space"
```

Comment:
I update indent style, if you don't like you can request change. In my opinion, this change will make us read code in GitHub easily by short indent. like below picture

<img width="1083" alt="image" src="https://github.com/coauthors/coauthors/assets/61593290/fefa2be0-37be-4c5e-87d2-ae5c152d31c0">